### PR TITLE
rec: 4.3.0 alpha3 prep

### PR DESCRIPTION
**Pull request by omoerbeek**
### Short description

alpha2 is tagged but will not be released due to a last-minute introduced bug :-(
So alpha3 it will be. 
Draft blog post in 
https://powerdnsblog.wordpress.com/?p=2905&preview=true

### Checklist

I have:
- [X] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Comment by omoerbeek** [comment]
https://builder.powerdns.com/#/builders/97/builds/978
and
https://builder.powerdns.com/#/buildrequests/121255

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 1]
```diff
@@ -1,6 +1,113 @@
...
+    :tags: Bug fixes
+    :pullreq: 8470
+
+    Rec: prime NS records of root-servers.net parent (.net)
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8463
+
+    Rec: Update CentOS 6 init script (None)
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/4.3.rst]: `(None)` is a bit weird here?

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 2]
```diff
@@ -1,6 +1,113 @@
...
 
+.. changelog::
+  :version: 4.3.0-alpha3
+  :released: 29th of October 2019
+
+  .. change::
+    :tags: Bug fixes
+    :pullreq: 8470
+
+    Rec: prime NS records of root-servers.net parent (.net)
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/4.3.rst]: I would remove the `rec:` prefix, I should update the script to do so.

File: pdns/recursordist/docs/changelog/4.3.rst [hunk 3]
```diff
@@ -1,6 +1,113 @@
...
+    :tags: Bug fixes
+    :pullreq: 8317
+
+    Retry getrandom() on EINTR
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 8287
+
+    Docs: Add small description for pipe backend about distributor-threads (Donatas Abraitis)
```
Comment (rgacogne) [inline pdns/recursordist/docs/changelog/4.3.rst]: I usually remove documentation updates from the ChangeLog, but it can be argued both ways.

**Pull request merged**